src/pages/review/[id].tsx

```diff
@@ -0,0 +1,227 @@
+import { trpc } from "@/utils/trpc";
+import React from "react";
+import { useRouter } from "next/router";
+import NextError from "next/error";
+import Link from "next/link";
+
+export default function Review() {
```

Comment:
do you think we'll have a page at `deck/[id]` with a different purpose? Wondering if we just use that versus `review`

Reply:
Good question!  The idea was that "review" would be an execution of the deck.
Possibilities in the future
- `/deck/[id]` -> An overview of your deck where you can look at all the cards in a list view and edit/add
- `/card/[id]` -> An editing page for a card?

I noticed that you had a "new" page in your branch. That seems perfectly clear! `/edit/[id]` could certainly be embedded to make things work perfectly fine. I'm not particularly hung up on any particular naming convention, but switched to "review" to make the use a little more clear than "deck" which I figured could have multiple interpretations.

That being said, I'm down to change to anything that makes sense and can be expanded upon!
@nleech16 @tliebert Thoughts?